kore/test/Test/Kore/Step/Simplification/Integration.hs

```diff
@@ -521,14 +520,11 @@ test_simplificationIntegration =
                     { term = mkTop Mock.boolSort
                     , predicate = makeIffPredicate
                         (makeOrPredicate
-                            (makeInPredicate Mock.boolSort
-                                (mkCeil Mock.setSort Mock.cf)
-                                (mkCeil Mock.setSort Mock.cg)
-                            )
-                            (makeInPredicate_
-                                (mkCeil Mock.setSort Mock.cf)
-                                (asInternal (Set.fromList []))
+                            (makeAndPredicate
+                                (makeCeilPredicate Mock.boolSort Mock.cf)
+                                (makeCeilPredicate Mock.boolSort Mock.cg)
```

Comment:
I don't think this is right. It should be `makeCeilPredicate _ (mkAnd Mock.cf Mock.cg)`, otherwise there is nothing that constrains `cf` and `cg` with respect to each other.

Reply:
Isn't `\ceil(\and(\ceil(a), \ceil(b)))` equivalent to `\and(\ceil(a), \ceil(b))`?

Reply:
Yes, you're right: I mis-read the original as `\in(cf, cg)` not `\in(\ceil(cf), \ceil(cg))`.